code/modules/uplink/uplink_items.dm

```diff
@@ -1689,6 +1689,8 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	name = "Glue"
 	desc = "A cheap bottle of one use syndicate brand super glue. \
 			Use on any item to make it undroppable. \
+			Takes approximately thirty seconds to dry up once used. \
+			Sudden movement may cause it to dry up instantly. \
```

Comment:
I think we should be clear that it happens when dropped or moved in the inventory. We don't need to be _quite_ as IC here compared to other places since saying how it works is more important

Reply:
Would "When applied, will stick the item in either thirty seconds or instantly upon dropping or picking it up" work better?